wizard/cypress/fixtures/titanic.json

```diff
@@ -0,0 +1,343 @@
+{
```

Comment:
JSON-LD regularly changes when the standard evolves. Can you either 1) point to the original file or 2) write a unit test (Python or JS) that tests that the 2 are equal?

Note: this can be done in the next ticket.

Reply:
cypress seemingly doesn't allow for more than one fixtures folder and symlinks (even relative ones) within a repo are not platform independant. i'll look into other ways to make this work in another ticket.

Reply:
Thanks! Can you create the ticket?